src/runtime/rpc/rpc_endpoint.cc

```diff
@@ -1042,12 +1069,38 @@ class RPCClientSession : public RPCSession, public DeviceAPI {
   bool IsLocalSession() const final { return false; }
 
  private:
+  void RPCMaxTransferRemoteReturnValue(TVMArgs args) {
+    // Use args[1] as return value, args[0] is tcode
```

Comment:
nit: could also cite unpack logic in RPCWrappedFunc in src/runtime/rpc/rpc_module.cc

Reply:
done